--- origsrc/clutter-gtk-0.10.2/clutter-gtk/Makefile.am	2009-07-28 12:04:12.000000000 -0500
+++ src/clutter-gtk-0.10.2/clutter-gtk/Makefile.am	2009-09-08 00:44:58.226725500 -0500
@@ -52,7 +52,7 @@ GtkClutter-@CLUTTER_GTK_API_VERSION@.gir
 		--add-include-path=$(srcdir) --add-include=path=. \
 		--include=Clutter-1.0 \
 		--include=Gtk-2.0 \
-		--library=clutter-gtk-@CLUTTER_GTK_API_VERSION@ \
+		--library=libclutter-gtk-@CLUTTER_GTK_API_VERSION@.la \
 		--libtool="$(top_builddir)/libtool" \
 		--output $@ \
 		--pkg gobject-2.0 \
