--- origsrc/clutter-gtk-0.10.8/clutter-gtk/Makefile.am	2010-09-16 08:44:00.000000000 -0500
+++ src/clutter-gtk-0.10.8/clutter-gtk/Makefile.am	2010-09-20 22:31:03.937787500 -0500
@@ -35,7 +35,7 @@ lib_LTLIBRARIES = libclutter-gtk-@CLUTTE
 # please, keep the list sorted alphabetically
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_SOURCES = $(sources_c)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LIBADD = $(CLUTTER_LIBS) $(GTK_LIBS)
-libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LDFLAGS = $(CLUTTER_LT_LDFLAGS)
+libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_LDFLAGS = $(CLUTTER_GTK_LT_LDFLAGS)
 libclutter_gtk_@CLUTTER_GTK_API_VERSION@_la_CFLAGS = $(MAINTAINER_CFLAGS) $(GTK_CFLAGS) $(CLUTTER_CFLAGS) 
 
 cluttergtkheadersdir = $(includedir)/clutter-1.0/clutter-gtk
--- origsrc/clutter-gtk-0.10.8/configure.ac	2010-09-16 08:49:08.000000000 -0500
+++ src/clutter-gtk-0.10.8/configure.ac	2010-09-20 22:30:51.829728700 -0500
@@ -43,7 +43,7 @@ CLUTTER_GTK_LT_CURRENT=lt_current
 CLUTTER_GTK_LT_REV=lt_revision
 CLUTTER_GTK_LT_AGE=lt_age
 CLUTTER_GTK_LT_VERSION="$CLUTTER_GTK_LT_CURRENT:$CLUTTER_GTK_LT_REV:$CLUTTER_GTK_LT_AGE"
-CLUTTER_GTK_LT_LDFLAGS="-version-info $CLUTTER_GTK_LT_VERSION"
+CLUTTER_GTK_LT_LDFLAGS="-version-info $CLUTTER_GTK_LT_VERSION -no-undefined"
 
 AC_SUBST(CLUTTER_GTK_LT_VERSION)
 AC_SUBST(CLUTTER_GTK_LT_LDFLAGS)
